docs/index.html

```diff
@@ -262,8 +262,6 @@
     </script>
     <!-- docsify (latest v4.x.x)-->
     <script src="https://cdn.jsdelivr.net/npm/docsify@4/lib/docsify.min.js"></script>
-    <!-- docsify-tabs (latest v1.x.x) -->
-    <script src="https://cdn.jsdelivr.net/npm/docsify-tabs@1"></script>
```

Comment:
for this file there is to remove the customization too, lines 159-165

https://github.com/freeCodeCamp/freeCodeCamp/blob/d0e816207b2d9e2f3fa2b8e0eda10edf4dcbfe3e/docs/index.html#L159-L165

Reply:
<img width="926" alt="image" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/79132440/e42d8bd0-97b6-4d47-9b52-d2e16695b9dc">
Is this okay so there's no conflict in headings?

Reply:
Removed